linkerd/distribute/src/service/first.rs

```diff
@@ -0,0 +1,80 @@
+use crate::keys::ServiceKeys;
+use linkerd_stack::{NewService, Service};
+use std::task::{Context, Poll};
+
+#[derive(Debug)]
+pub struct FirstAvailableSelection<S> {
```

Comment:
Not actually public:

```suggestion
pub(crate) struct FirstAvailableSelection<S> {
```

Reply:
Done